packages/ts-moose-lib/src/clients/redisClient.ts

```diff
@@ -187,7 +187,10 @@ export class MooseCache {
 
       // Use provided TTL or default to 1 hour
       const ttl = ttlSeconds ?? 3600;
-      await this.client.setEx(prefixedKey, ttl, stringValue);
+      return await this.client.set(prefixedKey, stringValue, {
+        EX: ttl,
+        GET: true,
```

Comment:
add a line specifying what the return value is? i believe it is the old value?

Reply:
Good call, added docs